index.html

```diff
@@ -145,11 +143,6 @@ <h3>Deprecated</h3>
       </section>
     </section>
 
-    <section id="glossary">
-      <h2>Important Terms</h2>
-      <div data-include="common/terms.html" data-oninclude="restrictReferences"></div>
-    </section>
-
```

Comment:
That seems a separate thing, but I'm curious where it was discussed to remove the terms?
(i agree it's maybe not important to have them here and probably that duplicates content from other spec)

Reply:
- Other (non-aria) specs need to reference some of these terms
- if they are defined in multiple places which do they reference?
- xrefs now deals with references to terms in other specs